client/main.lua

```diff
@@ -110,92 +98,79 @@ CreateThread(function()
-                                lib.showTextUI(Lang:t("text.store_helicopters"))
-                            else
-                                lib.showTextUI(Lang:t("text.helicopters"))
-                            end
-                        end
-                         
-                        local function onExit()
-                            lib.hideTextUI()
-                        end
-
-                        local function inside()
-                            if IsPedInAnyVehicle(cache.ped, false) then
-                                if IsControlJustReleased(0, 38) then
-                                    if IsPedInAnyVehicle(cache.ped, false) then
-                                        DeleteVehicle(GetVehiclePedIsIn(cache.ped, false))
-                                    end
-                                end
-                            else
-                                if IsControlJustReleased(0, 38) then
-                                    MenuHeliGarage()
-                                end
-                            end
-                        end
-                        
+                    DrawMarker(2, Config.Locations.heli.coords.x, Config.Locations.heli.coords.y, Config.Locations.heli.coords.z, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.3, 0.2, 0.15, 200, 200, 200, 222, false, false, 0, true, false, false, false)
+                    if #(pos - vector3(Config.Locations.heli.coords.x, Config.Locations.heli.coords.y, Config.Locations.heli.coords.z)) < 1.5 then
                         if not HeliZone then
                             HeliZone = lib.zones.box({
-                                coords = vec3(Config.Locations["heli"].coords.x, Config.Locations["heli"].coords.y, Config.Locations["heli"].coords.z),
+                                coords = vec3(Config.Locations.heli.coords.x, Config.Locations.heli.coords.y, Config.Locations.heli.coords.z),
                                 size = vec3(4, 4, 4),
                                 rotation = 267.49,
                                 debug = Config.DebugZones,
-                                inside = inside,
-                                onEnter = onEnter,
-                                onExit = onExit
+                                onEnter = function()
+                                    if IsPedInAnyVehicle(cache.ped, false) then
+                                        lib.showTextUI(Lang:t("text.store_helicopters"))
+                                    else
+                                        lib.showTextUI(Lang:t("text.helicopters"))
+                                    end
+                                end,
+                                onExit = function()
+                                    lib.hideTextUI()
+                                end,
+                                inside = function()
+                                    if IsPedInAnyVehicle(cache.ped, false) then
```

Comment:
Instead of `IsPedInAnyVehicle` everywhere you can just check if `cache.vehicle` is not false (aka `if cache.vehicle then`)